file-generating/ronn/template.md

```diff
@@ -0,0 +1,39 @@
+your-porgram-name(1) - quick title
```

Comment:
typo in program, also remove (1)

Reply:
The `(1)` is not really necessary, but it shows the user that they can replace the `1` with other numbers for different manpage types, like `1` = `Executable programs or shell commands`, and `6` = `Games` (for game manuals), etc...